src/StructuredLogger/StructuredLogger.csproj

```diff
@@ -19,16 +19,16 @@
     <PackageTags>MSBuild Log Logger Structure Structured</PackageTags>
   </PropertyGroup>
   <ItemGroup Condition=" '$(OS)' == 'Windows_NT' ">
-    <PackageReference Include="Nerdbank.GitVersioning" Version="$(NuGetVersionNerdbankGitVersioning)" PrivateAssets="all" />
+    <PackageReference Include="Nerdbank.GitVersioning" PrivateAssets="all" />
   </ItemGroup>
   <PropertyGroup>
     <DefineConstants>$(DefineConstants);NETCORE</DefineConstants>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Build.Framework" Version="$(MSBuildPackageVersion)" />
-    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="$(MSBuildPackageVersion)" />
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="all" />
-    <PackageReference Include="DotUtils.StreamUtils.Sources" Version="0.0.7" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.Build.Framework" />
+    <PackageReference Include="Microsoft.Build.Utilities.Core" />
+    <PackageReference Include="Microsoft.SourceLink.GitHub" />

```

Comment:
I notice that the NuGet package for MSBuild.StructuredLogger 2.2.317 is showing a dependency on Microsoft.SourceLink.GitHub 8.0.0, and I wonder if that was caused by the removal of PrivateAssets="all" here and whether that was intentional or not?

Reply:
Probably not intentional, I need to fix

Reply:
Published https://www.nuget.org/packages/MSBuild.StructuredLogger/2.2.337, thanks for spotting this

Reply:
Thanks

Reply:
My apologies. Sorry for that.

@KirillOsenkov I was looking through the <TargetFramework(s)> and noticed that most of them are net8.0 with a couple others. Is there any reason to not moving everything over to net8.0? (Or net9.0 in the near future?)

If we could move everything to net8.0 we could completely remove the SourceLink PackageReference: https://github.com/dotnet/sourcelink/releases/tag/8.0.0

Reply:
You might not need to actually change the TFMs to use the inbox SourceLink - just ensuring it's all built with the .NET 8.0 SDK might be sufficient?

Reply:
@Numpsy I'm not sure either. I err'ed on the side of it might be more beneficial to just move everything to net8.0?

The one thing I did forget about though is that this repo does generate some nuget packages, so to better support everyone out there it might not be practical to drop net472 or netstandard2.0?

I guess the only way to know for sure would be to test it out :) I'll see if I can do that this evening or in the near future because I've been wondering about that anyways.

Reply:
My request is to minimize contributions that don't solve real problems - maintaining this repo (among dozens of other repos I maintain) is already taking a huge toll on me, and PRs that look like help actually just cause more work for me in the future. 

You can read here for a recent example: 
https://github.com/KirillOsenkov/MSBuildStructuredLog/pull/801#issuecomment-2323676473

So ideally if you feel that there's a real problem with the app, file an issue, and for issues that I need help with, I can mark them as up-for-grabs, and then feel free to send PRs for those. 

My time and attention is incredibly scarce and valuable, so ideally I'd minimize interruptions and disruptions and pings as respect for my time and what I'm already doing for the community.

Thanks for your understanding!